File: wrappers/cpp/ipaship.cpp
#include "ipaship.hpp"
#include <iostream>

namespace IPAShip {
    Auditor::Auditor(const std::string& api_key) : api_key_(api_key) {}
    void Auditor::audit(const std::string& file_path) {
        std::cout << "Auditing " << file_path << " via ipaship.com..." << std::endl;
    }
}

File: wrappers/python/ipaship.py
import requests

class IPAShip:
    def __init__(self, api_key):
        self.api_key = api_key
        self.base_url = 'https://ipaship.com/api'

    def audit(self, file_path):
        print(f"Auditing {file_path} via ipaship.com...")
        return {"status": "success"}